justfile

```diff
@@ -12,7 +12,10 @@ test-e2e token:
 
 # check static typing
 check-types:
-    poetry run mypy
+    poetry run mypy --python-version 3.12
+    poetry run mypy --python-version 3.11
+    poetry run mypy --python-version 3.10
+    poetry run mypy --python-version 3.9
```

Comment:
I found this useful for local dev but is less useful for CI since we already run the full gamut across 4 python versions in CI. maybe this should be a separate just command?